parser/sql/dialect/mysql/src/main/antlr4/imports/mysql/BaseRule.g4

```diff
@@ -17,7 +17,7 @@
 
 grammar BaseRule;
 
-import Symbol, Keyword, MySQLKeyword, Literals;
+import Comments, Symbol, Keyword, MySQLKeyword, Literals;
```

Comment:
Why import Comments rule, and not use it in BaseRule?

Reply:
Hi mentor, the comment file was not imported, resulting in comments in SQL that cannot be parsed.
Invert the annotation file into BaseRule, as other files have direct or indirect references to it.

Reply:
Thank you for your explaination.